variables.tf

```diff
@@ -59,9 +59,22 @@ variable "wait_till_timeout" {
 
 variable "access_key" {
   type        = string
-  description = "Access key used by the IBM Cloud Monitoring agent to communicate with the instance"
+  description = "Access key used by the IBM Cloud Monitoring agent to communicate with the instance. Either `access_key` or `existing_access_key_secret_name` is required."
```

Comment:
You should mention that the access key will be stored in a new secret on the cluster if passed